src/fklearn/causal/cate_learning/meta_learners.py

```diff
@@ -264,3 +260,195 @@ def p(new_df: pd.DataFrame) -> pd.DataFrame:
 causal_s_classification_learner.__doc__ += learner_return_docstring(
     "Causal S-Learner Classifier"
 )
+
+
+def _simulate_t_learner_treatment_effect(
+    df: pd.DataFrame,
+    learners: dict,
+    treatments: list,
+    control_name: str,
+    prediction_column: str,
+) -> pd.DataFrame:
+    """
+    Simulate the treatment effect for the T-Leaner.
+    """
+    control_fcn = learners[control_name]
+    control_conversion_probability = control_fcn(df)[prediction_column].values
```

Comment:
What conversion means here?

Reply:
It is the output of the model. It should be the probability of conversion, right?